### ISSUE TITLE
BotConfig example

### ISSUE THREAD

**Issue by alkurbatov** [davechurchill/commandcenter#18]
I have several suggestion regarding the bot's config:

a. BotConfigs has .txt extension by some reason while it is .json. Could it be renamed? The .txt extension confuses ides and disables syntax highlight, making it easier to make a mistake.

b. Rename BotConfig.txt into Botconfig.example.txt (or something like it). The motivation: this config is just a general example. As a user of this project I have several different settings, especially regarding map path and have to rebase the settings on each change.
The example should be renamed manually (or during build) into BotConfig.txt while BotConfig.txt must be ignored in .gitignore.
This approach allows to keep local changes outside of git control while still receive the latest updates. Just for convenience.

**Comment by davechurchill** [comment]
a. The reason I have the .txt extansion is because with UAlbertaBot when I had a .json file, I received a surprising number of emails / questions about what program you use to open it, etc. So I left it as a .txt file for it to be more obvious. The general computer / programming skill of many users is not that high.

b. I'll think about this, but again, people will skip some instructions, run the bot, and it will say that the config file is missing and ask questions about it. I agree with the principle of what you're saying, just not sure if this is the way I want to go about it

**Comment by alkurbatov** [comment]
>a. The reason I have the .txt extansion is because with UAlbertaBot when I had a .json file, I received a surprising number of emails / questions about what program you use to open it.
I see, thank you for clarifying this. 

Regarding #b it just cause some inconvenience during dev, i.e. I have to cast:
```bash
$ git stash
$ git fetch
$ git merge upstream/master
$ git stash pop
# Resolve conflicts in the config.
```

Also I should be careful to not commit the config accidentally.

**Comment by alkurbatov** [comment]
>people will skip some instructions, run the bot, and it will say that the config file is missing and ask questions about it.

We can workaround it by searching BotConfig.txt first and BotConfig.example.txt next :) But I believe there should be a better solution.

Who are those people who use the bot? They checkout the sources or use something like .exe in an archive? For the latter it is possible to make special command (or script) which pack everything with proper names.

**Comment by davechurchill** [comment]
I agree with you that a change should be made, just gonna think for a while about how to do it

**Comment by stolencatkarma** [comment]
you could change it to BotConfig.example.txt and if there isn't one at runtime just rename the file to BotConfig.txt and continue. This let's new people just run it and have it work.

my 2 cents.
